guide/daily-spending-wallet/backup-and-recovery/landing-page.md

```diff
@@ -52,32 +74,27 @@ Two main types of information need to be backed up: the user's wallet recovery d
    height = 453
 %}
 
-- The _[Recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }})_ is the fundamental information (consisting of 12 or 24 words) used to construct a wallet
-- The _Passphrase_ is an optional password for additional security
-- The _[Derivation path]({{ '/guide/glossary/#derivation-path' | relative_url }})_ includes information on how to construct the wallet
-- A _Fingerprint_ allows for verification that the wallet was reconstructed correctly after import
-- _Static channel backups_ contain all basic information about active Lightning channels
-- A _Channels database_ contains data on payments made in active Lightning channels
+### Recovery Phrase
 
-Users who lose both their wallet access and their backup permanently lose their bitcoin. Arguably, this creates one of the most challenging design problems within the space.
+The _[recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }})_ is the fundamental information (consisting of 12 or 24 words) used to construct a wallet. This can be encrypted and automatically backed up to the cloud, or manually backed up on paper by the user.
 
-Backups could be handled automatically by the software, manually by the user, or a mix of these two methods. Regardless of the type of scheme you decide to go with, it’s crucial to be explicit about how your product handles backups and private key management.
+### Channel State
 
-{% include tip/recommendation.html %}
+This contains data on payments made in active Lightning channels. Think of this as sort of like the most recent snapshot of all the activity in the user's Lightning channels. This is important to maintain. While it may be possible to recover without this data, the user has to trust that the LSP is acting in their best interest. This data must be backed up automatically to the cloud every time a payment is sent or received.
```

Comment:
It can also be encrypted by the users recovery phrase before being backed up to the cloud for additional security. Breez and Phoenix both offer this.